README.md

```diff
@@ -19,14 +19,19 @@ You will then need to install and activate the virtual environment:
 pipenv install
 pipenv shell
 ```
-
 Once the virtual environment has been activated, to check that the `metriq-client` API is working as expected, run the 
 following example script from `metriq-client/examples/metriq_hello.py`:
 
+You will need to create an environment variable METRIQ_CLIENT_API_KEY to store the Metriq API key. On Linux/Unix operating systems, you can run
```

Comment:
Make sure that references to variable names in text are properly enclosed in backticks. So METRIQ_CLIENT_API_KEY should be `METRIQ_CLIENT_API_KEY`